data/noiseprofiles.json

```diff
@@ -23,6 +23,19 @@
             {"name": "EOS-1Ds iso 1250", "iso": 1250, "a": [7.95278526496704e-05, 3.32459647343708e-05, 5.14955370294715e-05], "b": [5.09601367196328e-07, 5.3070173385365e-07, 6.62837426268686e-07]}
           ]
         },
+        {
+          "comment": "ixy 220f contributed by Chase Patterson",
+          "model": "Canon IXY 220F",
```

Comment:
```suggestion
          "model": "IXY 220F",
```

Reply:
Thanks!

Reply:
I'm curious why this was changed. The camera model is recognized as `Canon IXY 220F`. With the change, this profile can no longer be applied to images from the camera.

Here is a sample raw image if you'd like to reproduce (https://raw.pixls.us/getfile.php/6984/nice/Canon%20-%20Canon%20IXY%20220F%20-%2012bit%20CHDK%20ver.%201.6.1%20(4:3).DNG).

See the metadata panel from darktable with this image:
<img src="https://github.com/darktable-org/darktable/assets/4053498/2078ed66-cd46-4230-8487-1478416c1e2a" width="300">

Reply:
I will let @kmilos confirm, but once darktable-org/rawspeed#565 will be merged, the name will be IXY 220F.

Reply:
Correct. The change has now been merged, but not yet propagated to rawspeed stable branch nor the corresponding dt submodule yet (should happen for the next release).

In the meantime, one can easily add the lines to `cameras.xml` as in the PR.

Reply:
Ah, I see it adds an alias. Thank you!

Reply:
@chapatt This is now be working on dt master branch (need to update submodules of course) and nightly builds.

Reply:
Awesome! Thanks all!